### PR TITLE
converts the templates into forms

### PR DESCRIPTION
**Pull request by piplup55**
similar to https://github.com/Ryujinx/Ryujinx/pull/4068

### CODE REVIEW

**Review by MutantAura — APPROVED** [review]
LGTM. I'll wait for goffy to have his say :)

File: .github/ISSUE_TEMPLATE/game_compatibility.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+name: Game Compatibility
+description: Report a game's compatibility status on Ryujinx
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for game?
```
Comment (EmulationFanatic) [inline .github/ISSUE_TEMPLATE/game_compatibility.yml:6]: "Is there an existing issue for the game?"

File: .github/ISSUE_TEMPLATE/game_compatibility.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      description: Please search to see if an issue already exists for the bug you encountered.
```
Comment (EmulationFanatic) [inline .github/ISSUE_TEMPLATE/game_compatibility.yml:7]: "Please search to see if an issue/page already exists for the game you're reporting on."

File: .github/ISSUE_TEMPLATE/game_compatibility.yml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        - Add links to pull requests that fix issues described if/when applicable. 
+        - Add a zipped save file for easy access to any issues if needed, namely for situations that can only be accessed after several hours of gameplay.
+    validations:
+      required: true
+  - type: textarea
+    id: screenshot
+    attributes:
+      label: Screenshots
+      description: Add screenshots here (preferably on windowed mode) if the game is rendering.
+      placeholder: You can drag and drop the images on to the textarea
```
Comment (EmulationFanatic) [inline .github/ISSUE_TEMPLATE/game_compatibility.yml:70]: "Drag and drop the images/screenshots onto the text area."

File: .github/ISSUE_TEMPLATE/game_compatibility.yml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    validations:
+      required: false
+  - type: textarea
+    id: log
+    attributes:
+      label: Log File
+      description: Logs files can be found under "Logs" folder in Ryujinx program folder. 
+      placeholder: You can drag and drop the log on to the textarea
```
Comment (EmulationFanatic) [inline .github/ISSUE_TEMPLATE/game_compatibility.yml:78]: "Drag and drop the log file onto the text area."

File: .github/ISSUE_TEMPLATE/game_compatibility.yml [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+      label: Screenshots
+      description: Add screenshots here (preferably on windowed mode) if the game is rendering.
+      placeholder: You can drag and drop the images on to the textarea
+    validations:
+      required: false
+  - type: textarea
+    id: log
+    attributes:
+      label: Log File
+      description: Logs files can be found under "Logs" folder in Ryujinx program folder.
```
Comment (EmulationFanatic) [inline .github/ISSUE_TEMPLATE/game_compatibility.yml:77]: "Log files can be found under the "Logs" subfolder in the Ryujinx program folder."

**Review by EmulationFanatic — COMMENTED** [review]
Added my suggestions.

**Pull request merged**